venv/lib/python3.12/site-packages/anyio/_core/_sockets.py

```
@@ -0,0 +1,991 @@
+                event = Event()
+                tg.start_soon(try_connect, addr, event)
+                with move_on_after(happy_eyeballs_delay):
+                    await event.wait()
+
+        if connected_stream is None:
+            cause = (
+                oserrors[0]
+                if len(oserrors) == 1
+                else ExceptionGroup("multiple connection attempts failed", oserrors)
+            )
+            raise OSError("All connection attempts failed") from cause
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)